bunq/sdk/model/core/installation.py

```diff
@@ -0,0 +1,91 @@
+from abc import ABC
+
+from bunq.sdk.http.api_client import ApiClient
+from bunq.sdk.json import converter
+from bunq.sdk.model.core.bunq_model import BunqModel
+
+
+class Installation(BunqModel, ABC):
```

Comment:
I don't think ABC does anything if you don't mark any method as abstract

furthermore, a bit weird since the parent isn't abstract

Reply:
PyCharm complains (e.g., `Module must implement all abstract methods`) if I do not use `ABC`. It's weird because `BunqModel` does not contain any abstract methods... Only a `from_json` static method which needs to be implemented by inheriting classes according to PyCharm. 

What do you suggest? I could change the `@staticmethod` method to be a classmethod and then we no longer have to use `ABC`. I might then need to implement that method in the various classes that use it.

Reply:
marking concrete classes as abstract to satisfy pycharm is wrong.
either implement the methods that currently raise `NotImplementedError` in the child classes or remove that abstract method from the parent class

Reply:
I just removed the method. Thanks.